snapshotserver/config.go

```diff
@@ -50,7 +47,16 @@ func SetConfigDefaults() {
 	pflag.StringP("tempdir", "T", "", "Set temporary directory for snapshot files")
 	viper.SetDefault("tempdir", defaultTempDir)
 
-	pflag.BoolP("config", "C", false, fmt.Sprintf("Use a config file named '%s' located in either /etc/%s/, ~/.%s or ./)", appName, packageName, packageName))
+	pflag.IntP("connmaxlife", "x", 5, "Sets the maximum amount of time (Minutes) a connection may be reused")
```

Comment:
I don't really like assigning "-x", "-y", and "-z" to these command-line parameters -- it seems a little arbitrary.

Given that I don't think that we should really change them very often, can we leave off the short names ("x", etc) and just use the long names? I THINK that PFlag will support that if you just replace "x" with the empty string.

Also, I think that the default for this value (5) should be a constant up at the top of the file so it's obvious.